brainscore_language/submission/endpoints.py

```diff
@@ -24,13 +24,23 @@ def score(self, model_identifier: str, benchmark_identifier: str) -> Score:
 
 
 def run_scoring(args_dict: Dict[str, Union[str, List]]):
+    model_ids, benchmark_ids = resolve_models_benchmarks(args_dict)
+    
+    for benchmark in benchmark_ids:
+        for model in model_ids:
+            run_scoring_endpoint(domain="language", jenkins_id=args_dict["jenkins_id"],
+                                model_identifier=model, benchmark_identifier=benchmark,
+                                user_id=args_dict["user_id"], model_type="artificialsubject",
+                                public=args_dict["public"], competition=args_dict["competition"])
+
+def resolve_models_benchmarks(args_dict: Dict[str, Union[str, List]]):
     benchmarks, models = retrieve_models_and_benchmarks(args_dict)
 
-    run_scoring_endpoint(domain="language", jenkins_id=args_dict["jenkins_id"],
-                         models=models, benchmarks=benchmarks, user_id=args_dict["user_id"],
-                         model_type="artificialsubject", public=args_dict["public"],
-                         competition=args_dict["competition"])
-
+    model_ids = resolve_models(domain="language", models=models)
+    benchmark_ids = resolve_benchmarks(domain="language", benchmarks=benchmarks)
+    print("BS_NEW_MODELS=" + " ".join(model_ids))
+    print("BS_NEW_BENCHMARKS=" + " ".join(benchmark_ids))
+    return model_ids, benchmark_ids
```

Comment:
Suggest moving `resolve_models_benchmarks()` to `core`

Reply:
I agree. I moved it to Core in [6eb46a4](https://github.com/brain-score/core/pull/40/commits/6eb46a421cb7a86bd9b80599e2b077800e3b0786) and removed it from Language in [d05eaf2](https://github.com/brain-score/language/pull/204/commits/d05eaf282d301fca8473f0a0366afd034d85a824).